streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -1958,64 +1859,58 @@ public void shouldNotCheckpointForSuspendedRunningTaskWithSmallProgress() {
 
         task.suspend();
         task.postCommit(false);
-        EasyMock.verify(stateManager);
+
+        verify(stateManager).checkpoint(); // checkpoint should only be called once
```

Comment:
Is this comment necessary? If you want to make it more explicit use
```java
verify(stateManager, times(1)).checkpoint();
```
instead.

But I think, `verify(stateManager).checkpoint()` is clear enough.

Reply:
I moved the same comment from the setup to the verification, but you are correct, it is not needed, I shall remove it